facilitator/src/test_utils.rs

```diff
@@ -119,8 +118,8 @@ pub fn default_pha_signing_public_key() -> UnparsedPublicKey<Vec<u8>> {
 // the top of any test we want logs from.
 // https://docs.rs/env_logger/0.8.2/env_logger/#capturing-logs-in-tests
 pub fn log_init() {
-    let _ = env_logger::builder()
-        .filter_level(LevelFilter::Info)
-        .is_test(true)
-        .try_init();
+    // let _ = env_logger::builder()
```

Comment:
Need to figure out if this is still applicable with this.

Reply:
[This](https://github.com/slog-rs/slog/wiki/FAQ#my-logs-are-not-captured-in-unit-tests) might help?